# Genomic Location is Information #
## Towards Effectively Assignment of Metabolic Pathway by Bacterial Genes ##
An intrinsic problem with homology-based pathway prediction, the dominating strategy for pathway prediction now, is that the mapped pathways generally have unassigned enzymes, also called orphan enzymes, since homologous pathways are in general not identical, each having its unique enzymatic reactions and/or using unique enzyme-encoding genes. Various techniques have been investigated aiming to remedy the issue but with only limited success. We present here a new method, ORENGE, for assigning such ORphan ENzymes using fundamentally novel GEnomic location information based on our recent discovery. Specifically, we have recently demonstrated that the location of any gene in a bacterial genome is tightly constrained by the genomic locations of the other genes (called relevant genes) encoding the metabolic pathway(s) involving this gene. Thus the to-be-identified gene for each orphan enzyme typically tends to have a narrow genomic range when the majority of its relevant genes are already assigned. When this information is applied in conjunction with accurate prediction of enzyme-encoding genes, it can lead to reliable assignments of orphan enzymes at a genome scale. We have tested our program on all 89 (partially) assigned E. coli K12 metabolic pathways in KEGG. A self-validation analysis shows that the percentage of currently assigned E. coli genes, falling in the top 10 predictions for their reactions, was improved from 60% to 96% by our program compared to the state-of-the-art prediction tools. In addition, the program can accurately assign, on average, 62% and 60% of the orphan enzymes artificially created by randomly and repeatedly removing 5% and 10% of the assigned genes across the 89 pathways, respectively. Furthermore, we have demonstrated a considerably improved performance by our algorithm compared to the existing algorithms on 14 real orphan enzymes according to the latest metabolic network constructed by Palssonâ€™s group, which were evaluated using the predicted flux of reactions in the network and matching transcriptomic data.

**Citing us**: Qin Ma, Xin Chen, Xizeng Mao, Hanyuan Zhang, Ying Xu, Assignment of Orphan Enzymes in Bacterial Metabolic Pathways: An application of genomic location information, _Ready for submission_.

**Contact**: Any questions, problems, bugs are welcome and should be dumped to Qin Ma **<maqin@uga.edu>**